ui/app/first-time/disclaimer.js

```diff
@@ -40,16 +40,40 @@ DisclaimerScreen.prototype.render = function () {
 
         .markdown {
           font-family: Times New Roman;
+          overflow-x: hidden;
         }
         .markdown h1, .markdown h2, .markdown h3 {
           margin: 10px 0;
-          font-family: arial sans-serif;
           font-weight: bold;
         }
 
+        .markdown strong {
+          font-weight: bold;
+        }
+        .markdown em {
+          font-style: italic;
+        }
+
+        .markdown p {
+          margin: 10px 0;
+        }
+
+        .markdown a {
+          color: blue;
+        }
+
       `),
 
       h('div.markdown', {
+        onScroll: (e) => {
+          var object = e.currentTarget
+          var button = document.getElementById('agree')
+          if ((object.offsetHeight + object.scrollTop + 100 >= object.scrollHeight) && button.disabled) {
```

Comment:
This condition is so long it's hard to know what it means. I'd love to see some variables defined that makes it easy to understand.